docs/managing-dragonfly/cluster-mode.md

```diff
@@ -25,6 +33,149 @@ True
 b'bar'
 ```
 
-Note that your application code may be using a redis client that does not require cluster commands, in this case you don't need to specify the `--cluster_mode` flag.
+Note that your application code may be using a Redis client that does not require cluster commands,
+in this case you don't need to specify the `--cluster_mode` flag.
+
+In case you need to migrate data from Redis Cluster to Dragonfly you may find [Redis
+Riot](https://developer.redis.com/explore/riot/) helpful.
+
+## Multi Node Cluster
+
+Sometimes vertical scaling is not enough. In those cases, you'll need to set up a Dragonfly Cluster.
+
+A Dragonfly Cluster is similar to a Redis Cluster:
+
+* Multiple Dragonfly servers participate in a single logical data store
+* It provides all cluster-related commands required by Redis client libraries
+* It [distributes keys](https://redis.io/docs/reference/cluster-spec/#key-distribution-model) in the
+  same way Redis does
+* It supports [hash tags](https://redis.io/docs/reference/cluster-spec/#hash-tags)
+
+**Any client-side code that uses Redis Cluster should be able to migrate to Dragonfly Cluster with
+no changes.**
+
+However, setting up and managing a Dragonfly Cluster is different from managing a Redis Cluster.
```

Comment:
I thing we should write about this in the second sentence - Dragonfly Cluster is similar to Redis Cluster in all client facing behaviour but it does not self managed as Redis Cluster in which nodes communicate with each other to discover cluster setup and state